cmd/aries-agentd/startcmd/start_test.go

```diff
@@ -104,7 +104,7 @@ func TestStartAriesDRequests(t *testing.T) {
 
 	go func() {
 		parameters := &agentParameters{&HTTPServer{}, testHostURL, testInboundHostURL,
-			"", path, []string{}}
+			"", path, "", []string{}}
```

Comment:
Not directly related to this PR - But it's better to initialize using field names. If we don't need to initialize, we can just ignore the field. Without name, we need to pass values sequentially.